.github/workflows/pr-validation.yml

```diff
@@ -29,3 +30,11 @@ jobs:
 
       - name: Run tests
         run: dotnet test
+
+  test-linux: #runs only P0 tests on linux
+    runs-on: ubuntu-latest
+    needs: build
+    steps:
+      - uses: actions/checkout@v2
+      - name: Run tests
+        run: dotnet test --filter Priority=0
```

Comment:
Why only run p0 tests? Are there other p faillures?

Reply:
Slightly embarrassed, but I had assumed there were many more tests beyond P0 and that there would certainly be failures. There's 21 more tests and they all pass so adding them to the workflow.